src/main/java/homework2/StringConcatenate2.java

```diff
@@ -0,0 +1,10 @@
+package homework2;
+
+public class StringConcatenate2 {
+
+    public static String concatenateAndRemoveP(String firstString, String secondString) {
+        String concatenate = firstString.concat(secondString);
+        String removePLetter = concatenate.replaceAll("([p,P])", "");
```

Comment:
You can return this expression instead of assigning it to the temporary variable "removePLetter". Also, the same comment for concatenate variable. all of this can be simplified to a one-line return without separate variables

Reply:
thanks, done